config/selectionCfg_ETau_Legacy2017_syst.cfg

```diff
@@ -886,15 +886,15 @@ DYclass_jesDown_11           = VBFloose_jesDown_11 , mpp_dy_v2_jesDown_11
 ## multiple weights are passed as list and are multiplied together
 ## NOTE: no weight is applied for data (the simple Fill() is used)
 [selectionWeights]
-baseline  = MC_weight, PUReweight, PUjetID_SF, L1pref_weight, prescaleWeight, trigSF, IdAndIsoAndFakeSF_deep_pt, DYscale_MTT
+baseline  = MC_weight, PUReweight, PUjetID_SF, L1pref_weight, prescaleWeight, trigSF, IdAndIsoAndFakeSF_deep_pt, DYscale_MTT, bTagweightReshape
```

Comment:
Just a general comment: why are we applying this weight also to the baseline where in the past we did not apply any btagweight?

Reply:
You're right, I keep forgetting. The QCD part instead seems ok to me.